tests/test_countgraph.py

```diff
@@ -467,24 +469,25 @@ def test_get_kmer_counts_too_short():
     hi = khmer.Countgraph(6, 1e6, 2)
 
     hi.consume("AAAAAA")
-    counts = hi.get_kmer_counts("A")
-    assert len(counts) == 0
+    with pytest.raises(ValueError):
+        counts = hi.get_kmer_counts("A")
 
 
 def test_get_kmer_hashes_too_short():
     hi = khmer.Countgraph(6, 1e6, 2)
 
     hi.consume("AAAAAA")
-    hashes = hi.get_kmer_hashes("A")
-    assert len(hashes) == 0
+    with pytest.raises(ValueError):
+        hashes = hi.get_kmer_hashes("A")
 
 
 def test_get_kmers_too_short():
     hi = khmer.Countgraph(6, 1e6, 2)
 
     hi.consume("AAAAAA")
-    kmers = hi.get_kmers("A")
-    assert len(kmers) == 0
+
+    with pytest.raises(ValueError):
+        kmers = hi.get_kmers("A")
```

Comment:
I like how these tests are raising `ValueError` instead of quietly returning empty lists (way easier to catch misbehaving code...)